rest/server_context.go

```diff
@@ -968,6 +967,8 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(ctx context.Context, config
 		if dbInitDoneChan != nil {
 			initError := <-dbInitDoneChan
 			if initError != nil {
+				// report error in building/creating indexes
+				dbcontext.DatabaseStartupError = db.NewDatabaseError(db.DatabaseInitialisationIndexError)
```

Comment:
Stick with US spelling for consistency - find/replace since there are other usages elsewhere from previous PR

```suggestion
				dbcontext.DatabaseStartupError = db.NewDatabaseError(db.DatabaseInitializationIndexError)
```